components/service/glean/scripts/sdk_generator.gradle

```diff
@@ -74,49 +74,57 @@ preBuild.dependsOn(createBuildDir)
 preBuild.finalizedBy("build_envs")
 
 // Generate the Metrics API
-ext.gleanGenerateMetricsAPI = { variant ->
-    def sourceOutputDir =  "$buildDir/telemetry/src/${variant.name}/kotlin"
-    def generateKotlinAPI = task("generateMetricsSourceFor${variant.name.capitalize()}", type: Exec) {
-        description = "Generate the Kotlin code for the Metrics API"
-
-        workingDir  rootDir
-        // Note that the command line is OS dependant: on linux/mac is Miniconda3/bin/python.
-        if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-            commandLine "$buildDir/bootstrap/Miniconda3/python"
-        } else {
-            commandLine "$buildDir/bootstrap/Miniconda3/bin/python"
-        }
+ext.gleanGenerateMetricsAPI = {
```

Comment:
This diff is less big than it seems -- it's mostly indenting to add the outer closure.